### PR TITLE
Windows 11 ADK USMT

### PR DESCRIPTION
**Pull request by robgrame**
Just make some changes to the page but my main reason is asking whether to have a specular page covering Windows 11 USMT migration supported scenarios. Unfortunately I cannot find anything specific to Windows 11 even though I am pretty sure Windows 11 ADK USMT is supposed to cover migration scenarios from Windows 10. Thanks

/roberto

## Why

- Closes #[Issue Number]

### CODE REVIEW

File: windows/deployment/usmt/usmt-requirements.md [hunk 1]
```diff
@@ -27,14 +27,15 @@ The following table lists the operating systems supported in USMT.
...
+|64-bit versions of Windows 11|✔️|✔️|
 
 > [!NOTE]
 > You can migrate a 32-bit operating system to a 64-bit operating system. However, you cannot migrate a 64-bit operating system to a 32-bit operating system.
 
 ## Unsupported scenarios
 
 - USMT doesn't support any of the Windows Server® operating systems.
-- USMT for Windows 10 shouldn't be used for migrating between previous versions of Windows. USMT for Windows 10 is only meant to migrate to Windows 10 or between Windows 10 versions. For more information about previous releases of the USMT tools, see [User State Migration Tool (USMT) overview](/previous-versions/windows/hh825227(v=win.10)).
+- USMT for Windows 10/11 shouldn't be used for migrating between previous versions of Windows. USMT for Windows 10 is only meant to migrate to Windows 10 or between Windows 10 versions. For more information about previous releases of the USMT tools, see [User State Migration Tool (USMT) overview](/previous-versions/windows/hh825227(v=win.10)).
```
Comment (JohanFreelancer9) [inline windows/deployment/usmt/usmt-requirements.md:38]:
```suggestion
- USMT for Windows 10 and Windows 11 shouldn't be used for migrating between previous versions of Windows. USMT for Windows 10 is only meant to migrate to Windows 10 or between Windows 10 versions. For more information about previous releases of the USMT tools, see [User State Migration Tool (USMT) overview](/previous-versions/windows/hh825227(v=win.10)).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @robgrame, kindly commit the suggested change. Many thanks.

**Comment by frankroj** [comment]
Need to confirm that USMT is supported to migrate from Windows 10 to Windows 11, and then reword article accordingly.

**Comment by frankroj** [comment]
@robgrame in general all supported scenarios for Windows 10 USMT is also supported for Windows 11 USMT. Updating these articles are on our to do list but it may be some time before we get to it. The one scenario I need to confirm is migrating from Windows 10 to Windows 11, which I am pretty sure is supported.

File: windows/deployment/usmt/usmt-requirements.md (hunk 1 above)
Comment (robgrame) [reply windows/deployment/usmt/usmt-requirements.md:38]: Hi @JohanFreelancer9 . 
From my point of view we are not clarifying yet the supportability to use USMT to migrate profiles from Windows 10 to Windows 11.
We should clearly state if Windows 11 ADK USMT is supposed (read supported) to migrate Windows 10 profiles to Windows 11 or it's not supported.
Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by frankroj** [comment]
Confirming with PMs what exactly is supported with Windows 11.

**Comment by frankroj** [comment]
Thank you @robgrame for the contribution. Although the contribution is good and valid, in the next month or two, I am going to be working on a greater project to update the USMT documentation with regards to Windows 11 support. I will make the changes as part of that project.

**Pull request closed without merging**